lib/vsc/utils/script_tools.py

```diff
@@ -129,6 +131,7 @@ def __init__(self, options, run_prologue=True, excepthook=None, **kwargs):
 
         self.nagios_reporter = None
         self.lockfile = None
+        self.monitorclass = monitorclass
```

Comment:
same as with the reporter, why not use a class constant?

Reply:
fixed in https://github.com/hpcugent/vsc-utils/pull/81/commits/507804c29499c399c97d2809de214eb64f778473